source/reporting/index.html.md.erb

```diff
@@ -0,0 +1,120 @@
+---
+title: Reporting 
+weight: 70
+---
+
+# Reporting 
+
+You can extract reporting information from GOV.UK Pay manually, or via the
+API.
+
+## Manual reporting
+
+You can sign in to your [GOV.UK Pay admin
+account](https://selfservice.payments.service.gov.uk/login) and export
+transaction information in a CSV file. 
+
+For a given account, select __Transactions__ in the __Dashboard__ page.
+
+You can filter transactions using the available search criteria, and then
+select __Download all transaction details as a CSV file__. 
+
+You can process CSV files using spreadsheet software.
+
+## Reporting via the GOV.UK Pay API 
+
+Before reading this section, you should read the section describing [Making
+payments](/making_payments). 
+
+You can develop software to get data from the GOV.UK Pay API and import it into 
```

Comment:
Partially contradicting myself earlier, I think this now has too much of an emphasis on software development, which isn’t strictly necessary to use the API for one-off things.

Perhaps it could say something like this (which also makes the example more concrete): “You can get data from the GOV.UK Pay API in a format that’s suitable for automatic processing. You could, for example, develop software to periodically query the GOV.UK Pay API for payment data and import it into your finance system. This could remove routine manual operations from your workflow and lead to significant cost savings for your organisation”

Reply:
> Perhaps it could say something like this (which also makes the example more concrete): “You can get data from the GOV.UK Pay API in a format that’s suitable for automatic processing. You could, for example, develop software to periodically query the GOV.UK Pay API for payment data and import it into your finance system. This could remove routine manual operations from your workflow and lead to significant cost savings for your organisation”

Makes sense, I'll go with that as a base for a new version.

Reply:
Just to be clear about what I meant by that, I'll basically use that content you've just written but restyle it a bit to fit it with GDS copy/style guidance.